float64 velocity
float64 command_voltage
